docs/topics/request-response.rst

```diff
@@ -507,7 +507,13 @@ Response objects
 
     .. attribute:: Response.headers
 
-        A dictionary-like object which contains the response headers.
+        A dictionary-like object which contains the response headers. Values can
+        be accessed from the :class:`headers` object by using :meth:`get` to
```

Comment:
yup, that's not resolving to a proper link -- no need to use special markup here, just headers is good.